diff --git a/drivers/bluetooth/Kconfig b/drivers/bluetooth/Kconfig
index e30707405..58c83c160 100644
--- a/drivers/bluetooth/Kconfig
+++ b/drivers/bluetooth/Kconfig
@@ -4,23 +4,23 @@ menu "Bluetooth device drivers"
 	depends on BT
 
 config BT_INTEL
-	tristate
+	tristate "Unhidden"
 	select REGMAP
 
 config BT_BCM
-	tristate
+	tristate "Unhidden"
 	select FW_LOADER
 
 config BT_RTL
-	tristate
+	tristate "Unhidden"
 	select FW_LOADER
 
 config BT_QCA
-	tristate
+	tristate "Unhidden"
 	select FW_LOADER
 
 config BT_MTK
-	tristate
+	tristate "Unhidden"
 	select FW_LOADER
 
 config BT_HCIBTUSB
@@ -106,7 +106,7 @@ config BT_HCIUART
 	  kernel or say M to compile it as module (hci_uart).
 
 config BT_HCIUART_SERDEV
-	bool
+	bool "Unhidden"
 	depends on SERIAL_DEV_BUS && BT_HCIUART
 	default y
 
@@ -423,7 +423,7 @@ config BT_QCOMSMD
 	  kernel or say M to compile as a module.
 
 config BT_HCIRSI
-	tristate
+	tristate "Unhidden"
 	help
 	  Redpine BT driver.
 	  This driver handles BT traffic from upper layers and pass
diff --git a/drivers/media/Kconfig b/drivers/media/Kconfig
index 351c1e8cc..fe3a999bd 100644
--- a/drivers/media/Kconfig
+++ b/drivers/media/Kconfig
@@ -175,7 +175,7 @@ config MEDIA_CONTROLLER
 #
 
 config DVB_CORE
-	tristate
+	tristate "Unhidden"
 	depends on MEDIA_DIGITAL_TV_SUPPORT
 	depends on (I2C || I2C=n)
 	default MEDIA_DIGITAL_TV_SUPPORT
@@ -239,14 +239,14 @@ endmenu
 #
 
 config MEDIA_HIDE_ANCILLARY_SUBDRV
-	bool
+	bool "Unhidden"
 	depends on MEDIA_SUBDRV_AUTOSELECT && !COMPILE_TEST && !EXPERT
 	default y
 
 menu "Media ancillary drivers"
 
 config MEDIA_ATTACH
-	bool
+	bool "Unhidden"
 	depends on MEDIA_ANALOG_TV_SUPPORT || MEDIA_DIGITAL_TV_SUPPORT || MEDIA_RADIO_SUPPORT
 	depends on MODULES
 	default MODULES
diff --git a/drivers/net/wireless/ath/Kconfig b/drivers/net/wireless/ath/Kconfig
index d88edbf1b..82158d7b1 100644
--- a/drivers/net/wireless/ath/Kconfig
+++ b/drivers/net/wireless/ath/Kconfig
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: ISC
 config ATH_COMMON
-	tristate
+	tristate "Unhidden"
 
 config WLAN_VENDOR_ATH
 	bool "Atheros/Qualcomm devices"
diff --git a/drivers/net/wireless/ath/ath9k/Kconfig b/drivers/net/wireless/ath/ath9k/Kconfig
index e150d82ed..14938cfd9 100644
--- a/drivers/net/wireless/ath/ath9k/Kconfig
+++ b/drivers/net/wireless/ath/ath9k/Kconfig
@@ -1,11 +1,11 @@
 # SPDX-License-Identifier: ISC
 config ATH9K_HW
-	tristate
+	tristate "Unhidden"
 config ATH9K_COMMON
-	tristate
+	tristate "Unhidden"
 	select ATH_COMMON
 config ATH9K_COMMON_DEBUG
-	bool
+	bool "Unhidden"
 config ATH9K_DFS_DEBUGFS
 	def_bool y
 	depends on ATH9K_DEBUGFS && ATH9K_DFS_CERTIFIED
diff --git a/drivers/net/wireless/ath/carl9170/Kconfig b/drivers/net/wireless/ath/carl9170/Kconfig
index ba9bea793..d72c8f914 100644
--- a/drivers/net/wireless/ath/carl9170/Kconfig
+++ b/drivers/net/wireless/ath/carl9170/Kconfig
@@ -34,7 +34,7 @@ config CARL9170_DEBUGFS
 	  Say N.
 
 config CARL9170_WPC
-	bool
+	bool "Unhidden"
 	depends on CARL9170 && (INPUT = y || INPUT = CARL9170)
 	default y
 
