diff --git a/modules.bzl b/modules.bzl
index 52d92225a..efae9a77f 100644
--- a/modules.bzl
+++ b/modules.bzl
@@ -8,6 +8,23 @@ This module contains a full list of kernel modules
 
 _COMMON_GKI_MODULES_LIST = [
     # keep sorted
+    "net/wireless/cfg80211.ko",
+    "net/mac80211/mac80211.ko",
+    "drivers/net/wireless/mediatek/mt76/mt76.ko",
+    "drivers/net/wireless/mediatek/mt76/mt76-usb.ko",
+    "drivers/net/wireless/mediatek/mt76/mt76x02-lib.ko",
+    "drivers/net/wireless/mediatek/mt76/mt76x02-usb.ko",
+    "drivers/net/wireless/mediatek/mt76/mt76x2/mt76x2-common.ko",
+    "drivers/net/wireless/mediatek/mt76/mt76x2/mt76x2u.ko",
+    "drivers/net/wireless/ath/ath.ko",
+    "drivers/net/wireless/ath/ath9k/ath9k_common.ko",
+    "drivers/net/wireless/ath/ath9k/ath9k_htc.ko",
+    "drivers/net/wireless/ath/ath9k/ath9k_hw.ko",
+    "drivers/bluetooth/bfusb.ko",
+    "drivers/bluetooth/btintel.ko",
+    "drivers/bluetooth/btrtl.ko",
+    "drivers/bluetooth/btusb.ko",
+    "drivers/net/wireless/ath/carl9170/carl9170.ko",
     "drivers/block/zram/zram.ko",
     "drivers/bluetooth/btbcm.ko",
     "drivers/bluetooth/btqca.ko",
