diff --git a/modules.bzl b/modules.bzl
index 52d92225a..ad2f92db3 100644
--- a/modules.bzl
+++ b/modules.bzl
@@ -8,6 +8,14 @@ This module contains a full list of kernel modules
 
 _COMMON_GKI_MODULES_LIST = [
     # keep sorted
+    "drivers/net/wireless/mediatek/mt76/mt76.ko",
+    "drivers/net/wireless/mediatek/mt76/mt76-usb.ko",
+    "drivers/net/wireless/mediatek/mt76/mt76x02-lib.ko",
+    "drivers/net/wireless/mediatek/mt76/mt76x02-usb.ko",
+    "drivers/net/wireless/mediatek/mt76/mt76x2/mt76x2-common.ko",
+    "drivers/net/wireless/mediatek/mt76/mt76x2/mt76x2u.ko",
+    "net/wireless/cfg80211.ko",
+    "net/mac80211/mac80211.ko",
     "drivers/block/zram/zram.ko",
     "drivers/bluetooth/btbcm.ko",
     "drivers/bluetooth/btqca.ko",
